.github/workflows/release-published.yml

```diff
@@ -1,80 +1,52 @@
 name: Release Extension to Sonatype
 
 on:
+  workflow_dispatch:
   release:
     types: [published]
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'Release Tag'
-        required: true
 
 jobs:
-  setup:
-    name: Setup
-    runs-on: ubuntu-latest
-    outputs:
-      tag: ${{ steps.collect-data.outputs.tag }}
-      version: ${{ steps.collect-data.outputs.version }}
-    steps:
-      - name: Collect Data
-        id: collect-data
-        uses: actions/github-script@v6
-        with:
-          script: |
-            let tag;
-            if (context.payload.inputs) {
-              tag = context.payload.inputs.tag;
-            } else {
-              tag = context.payload.release.tag_name;
-            }
-
-            let version = tag.replace(/^v/, "");
-
-            core.setOutput("tag", tag);
-            core.setOutput("version", version);
-
-      - run: |
-          echo "Publishing version ${{ steps.collect-data.outputs.version }} from ${{ steps.collect-data.outputs.tag }}"
+  release:
+    uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.5.2
+    secrets: inherit
 
-  deploy_maven:
-    name: Deploy to Maven
-    needs: [ setup ]
-    runs-on: ubuntu-latest
+  deploy_xsd:
+    name: Upload xsds
+    runs-on: ubuntu-20.04
     steps:
-      - name: Download Release Artifacts
-        uses: robinraju/release-downloader@v1.4
-        with:
-          tag: "${{ needs.setup.outputs.tag }}"
-          filename: "*"
-          out-file-path: "."
-
-      - name: Set up Java for publishing to Maven Central Repository
-        uses: actions/setup-java@v3
+      - name: Download liquibase mongodb xsd
+        uses: actions/checkout@v4
         with:
-          java-version: '17'
-          distribution: 'temurin'
-          server-id: sonatype-nexus-staging
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
+          # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-mongodb
+          repository: "liquibase/liquibase-mongodb"
 
-      - name: Publish to Maven Central
+      - name: Upload to s3
```

Comment:
@jandroav same thing here. I think its just copy/pasta from another extension.  We shouldn't need to upload XSDs

Reply:
fixed